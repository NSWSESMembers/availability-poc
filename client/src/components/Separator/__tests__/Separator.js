import React from 'react';
import renderer from 'react-test-renderer';

import { Separator } from '../';

test('renders correctly', () => {
  const tree = renderer.create(
    <Separator />,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
