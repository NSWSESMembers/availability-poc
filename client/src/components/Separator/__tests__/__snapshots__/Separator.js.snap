// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#EEE",
      "height": 0.5,
      "width": "100%",
    }
  }
/>
`;
